presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -117,15 +117,15 @@ public QueryManagerConfig setInitialHashPartitions(int initialHashPartitions)
     }
 
     @NotNull
-    public Duration getMaxQueryAge()
+    public Duration getMinQueryExpireAge()
     {
-        return maxQueryAge;
+        return minQueryExpireAge;
     }
 
-    @Config("query.max-age")
-    public QueryManagerConfig setMaxQueryAge(Duration maxQueryAge)
+    @Config("query.min-expire-age")
```

Comment:
Assuming we want to make this change, the correct way is like this:
```java
@LegacyConfig("query.max-age")
@Config("query.min-expire-age")
```
This will keep the old config name working, but print a warning if the old name is used. See `FeaturesConfig` for another example.